### DISCUSSION TITLE
Test request using multipart/form-data

### DISCUSSION THREAD

**Discussion by DocKDE** [tokio-rs/axum#2404] [Q&A]
### Summary

Hi! I'm in the process of upgrading to 0.7 and have to write new tests. I'd like to test sending requests with a multipart/form-data body. I could not find anything about this in the examples and am at a loss whether I can use the `reqwest` crate for this somehow. The axum request builder does not seem to provide for this. Is there some blessed way to do this or do I have to write the body by hand?

### axum version

0.7.1

**Comment by davidpdrsn** [comment]
You probably have to wait until reqwest supports http 1.0
